templates/default/profile.erb

```diff
@@ -1,48 +1,59 @@
-RDECK_HOME=<%=@rundeck[:basedir]%>
-export RDECK_HOME
-
-RDECK_BASE=<%=@rundeck[:basedir]%>
-export RDECK_BASE
-
-JAVA_CMD=java
-RUNDECK_TEMPDIR=<%=@rundeck[:tempdir]%>
-
+RDECK_BASE="<%=@rundeck[:basedir]%>"
```

Comment:
I followed the rundeck upgrade docs http://rundeck.org/docs/upgrading/index.html. If you install rundeck manually you would get the profile file same as the one I put as a template. The changes that are required on use both 2.6.x (2.6.9 and onwards) and 2.7.x are included in the template.